airbyte-integrations/connectors/source-callrail/source_callrail/callrail.yaml

```diff
@@ -0,0 +1,116 @@
+version: "0.1.0"
+
+definitions:
+  page_size: 100
+  step: "100d"
+
+  schema_loader:
+    type: JsonSchema
+    file_path: "./source_callrail/schemas/{{ options['name'] }}.json"
+
+  requester:
+    type: HttpRequester
+    name: "{{ options['name'] }}"
+    http_method: "GET"
+    authenticator:
+      type: ApiKeyAuthenticator
+      header: "Authorization"
+      api_token: "Token token={{ config.api_key }}"
+
+  stream_slicer:
+    type: "DatetimeStreamSlicer"
+    start_datetime:
+      datetime: "{{ config.start_date }}"
+      datetime_format: "%Y-%m-%d"
+    end_datetime:
+      datetime: "{{ today_utc() }}"
+      datetime_format: "%Y-%m-%d"
+    step: "*ref(definitions.step)"
+    cursor_field:  "{{ options.stream_cursor_field }}"
+    start_time_option:
+      field_name: "start_date"
+      inject_into: "request_parameter"
+    datetime_format: "%Y-%m-%dT%H:%M:%S.%f%z"
+
+  retriever:
+    type: SimpleRetriever
+    $options:
+      url_base: "https://api.callrail.com/v3/a/"
+    name: "{{ options['name'] }}"
+    primary_key: "{{ options['primary_key'] }}"
+    record_selector:
+      extractor:
+        type: DpathExtractor
+        field_pointer: [ "{{ options['name'] }}" ]
+    paginator:
+      type: NoPagination
```

Comment:
Please implement the correct pagination for this connector.

Reply:
I've added pagination in my latest commit with hash:  e105d10cb2f8af9fbc322139b1d6192152a7f219.